src/GridFieldOrderableRows.php

```diff
@@ -384,29 +383,25 @@ public function getManipulatedData(GridField $grid, SS_List $list)
                         $sortterm .= "$col $dir, ";
                     }
                 } else {
-                    $sortterm = $this->extraSortFields.', ';
+                    $sortterm = $this->extraSortFields . ', ';
                 }
             }
-            if ($list instanceof ArrayList) {
```

Comment:
Since this is not backward compatible I thought it best to remove this `if` statement